linalg/slepc.cpp

```diff
@@ -62,8 +62,10 @@ SlepcEigenSolver::SlepcEigenSolver(MPI_Comm comm, const std::string &prefix)
 
 SlepcEigenSolver::~SlepcEigenSolver()
 {
-   if (VR) { delete VR; }
-   if (VC) { delete VC; }
+   delete VR;
+   delete VC;
+   VR = VC = NULL;
```

Comment:
No need to set to NULL in a destructor, since you are not going to reuse the object later.